### PR TITLE
feat: default filter showcase

### PR DESCRIPTION
**Pull request by 2fd**

### CODE REVIEW

File: src/modules/translation/languages/es.json [hunk 1]
```diff
@@ -382,7 +382,8 @@
     "projects_title": "Scene Pool",
     "no_projects": "It looks like we don't have any Scenes",
     "filters": {
-      "creator-contest-2019-12": "Creator Contest",
+      "creator-contest-2019-12": "Creator Contest (Submition)",
```
Comment (abarmat) [inline src/modules/translation/languages/es.json]: Submition should be Submission

Comment (2fd) [reply src/modules/translation/languages/es.json]: :+1:

File: src/modules/translation/languages/en.json [hunk 2]
```diff
@@ -379,7 +379,8 @@
     "projects_title": "Scene Pool",
     "no_projects": "It looks like we don't have any Scenes",
     "filters": {
-      "creator-contest-2019-12": "Creator Contest",
+      "creator-contest-2019-12": "Creator Contest (Submition)",
```
Comment (2fd) [inline src/modules/translation/languages/en.json]:
```suggestion
      "creator-contest-2019-12": "Creator Contest (Submission)",
```

File: src/modules/translation/languages/es.json (hunk 1 above)
Comment (2fd) [inline src/modules/translation/languages/es.json]:
```suggestion
      "creator-contest-2019-12": "Creator Contest (Submission)",
```

File: src/modules/translation/languages/zh.json [hunk 3]
```diff
@@ -374,7 +374,8 @@
     "projects_title": "Scene Pool",
     "no_projects": "It looks like we don't have any Scenes",
     "filters": {
-      "creator-contest-2019-12": "Creator Contest",
+      "creator-contest-2019-12": "Creator Contest (Submition)",
```
Comment (2fd) [inline src/modules/translation/languages/zh.json]:
```suggestion
      "creator-contest-2019-12": "Creator Contest (Submission)",
```

**Pull request merged**